src/Microsoft.AspNet.Razor/RazorResources.resx

```diff
@@ -422,4 +422,7 @@ Instead, wrap the contents of the block in "{{}}":
     <value>Inline markup blocks (e.g. @&lt;p&gt;content&lt;/p&gt;) must not appear in non-string tag helper attribute values.
  Expected a '{0}' attribute value, not a string.</value>
   </data>
+  <data name="RewriterError_EmptyTagHelperBoundAttribute" xml:space="preserve">
+    <value>Attribute '{0}' on tag helper element '{1}' is a '{2}' tag helper bound attribute with no value. Only '{3}' typed attributes can be without a value.</value>
```

Comment:
not understandable or helpful. do not mention alternatives (a `string` bound attribute for example) that are out of the Razor author's control. suggest `Invalid empty value for attribute '{0}' on tag helper element '{1}'.  Tag helper bound attributes of type '{2}' must have a value.`